UI/data/locale/en-US.ini

```diff
@@ -626,6 +629,7 @@ Basic.Main.StartRecording="Start Recording"
 Basic.Main.StartReplayBuffer="Start Replay Buffer"
 Basic.Main.SaveReplay="Save Replay"
 Basic.Main.StartStreaming="Start Streaming"
+Basic.Main.StartBroadcast="GO LIVE"
```

Comment:
I guess it’s intentional that this is all caps?

Reply:
They're probably trying to imitate their website.
![Screenshot_20210729-121745-258](https://user-images.githubusercontent.com/5058319/127475339-71c681a9-b78e-4122-bc33-8b6a96e76a71.png)